File: APP/src/app/model/bermuda.ts
import { Produto } from "./produto";

enum Ajuste {
    ELASTICO, BOTAO, AMARRACAO
}

enum Comprimento {
    CURTO, MEDIO, LONGO
}

export class Bermuda extends Produto {
    ajuste: Ajuste | null;
    comprimento: Comprimento | null;

    constructor() {
        super();
        this.ajuste = null;
        this.comprimento = null;
    }
}

File: JAVA/src/main/java/com/inf_112/trabalho_final/model/Bermuda.java
package com.inf_112.trabalho_final.model;

import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper = true)
public class Bermuda extends Produto {
    public enum Ajuste {
        ELASTICO, BOTAO, AMARRACAO
    }

    public enum Comprimento {
        CURTO, MEDIO, LONGO
    }

    private Ajuste ajuste;
    private Comprimento comprimento;
}
